test/node/html-render.js

```diff
@@ -54,185 +54,4 @@ describe('HTML Driver', function () {
       done();
     });
   });
-
-  it('should render a simple nested custom element as HTML', function (done) {
-    function myElement() {
-      return {
-        DOM: Rx.Observable.just(h('h3.myelementclass'))
-      };
-    }
-    function app() {
-      return {
-        DOM: Rx.Observable.just(h('div.test-element', [h('my-element')]))
-      };
-    }
-    let [requests, responses] = Cycle.run(app, {
-      DOM: makeHTMLDriver({'my-element': myElement})
-    });
-    responses.DOM.subscribe(html => {
-      assert.strictEqual(html,
-        '<div class="test-element">' +
-          '<h3 class="myelementclass"></h3>' +
-        '</div>'
-      );
-      done();
-    });
-  });
```

Comment:
Depends. That would be a unit test. But there are integration tests, which basically test "This feature works". But anyway, these tests here would be testing e.g. transposition together with the HTML rendering driver.